SonarQube.Client/ISonarQubeService.cs

```diff
@@ -127,8 +127,8 @@ public interface ISonarQubeService
         Task<ServerExclusions> GetServerExclusions(string projectKey, CancellationToken token);
 
         /// <summary>
-        /// Creates a new <see cref="IServerSentEventsSession"/> for the given <see cref="projectKey"/>
+        /// Creates a new <see cref="ISSEStream"/> for the given <see cref="projectKey"/>
```

Comment:
I started to get confused with the term "Session", so I decided to rename it to Stream